src/components/ActiveVideo/ActiveVideo.tsx

```diff
@@ -0,0 +1,57 @@
+import { ActiveVideoCapture, LivenessError } from '@onfido/active-video-capture'
+import { h, FunctionComponent } from 'preact'
+import { useState } from 'preact/hooks'
+import { localised } from '~locales'
+import type { WithLocalisedProps } from '~types/hocs'
+import type { StepComponentBaseProps } from '~types/routers'
+
+type Props = StepComponentBaseProps & WithLocalisedProps
+
+const ActiveVideo: FunctionComponent<Props> = (props) => {
+  const { nextStep } = props
+  const [videoPayload, setVideoPayload] = useState<Blob | null>(null)
+  const [success, setSuccess] = useState<boolean>(false)
+  // FIXME: Move to src/locales/en_US/en_US.json
+  const localisedText = {
+    avc_face_alignment: {
+      feedback_move_back: 'Move back',
+      feedback_move_closer: 'Move closer',
+      feedback_no_face_detected: 'Face not detected',
+      feedback_not_centered: 'Face not centered',
+      title: 'Position your face in the frame',
+    },
+    avc_face_capture: {
+      alert: {
+        timeout_body: 'You have up to 20 seconds to complete the recording',
```

Comment:
should we change this right away to 15? I know the component it self still allows 20 but we're gonna have to change that to match Android & iOS behaviour